documentation/reference/releases/2024-sept-3.0.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 2024-SEPT Release 3.0.0
+order: 20240919
+date: 2024-SEPT-19
+---
+
+# API Services Portal Release 3.0.0
+
+The API Services Portal has been updated with enhancements in the existing features.
+
+## Enhancements
+
+* The term 'Namespace' is now renamed to 'Gateway'.
+  * Across all our products, every mention of 'Namespace' (which refers to the container used to hold a collection of Kong Gateway Services and Routes, typically managed by a single team or business unit) has been updated to 'Gateway'.
+  * All existing Namespaces will be retained but will now be referred to as Gateways.
+  * The only remaining trace of Namespace is the `ns.` prefix required in tags for Gateway Service configuration (`tags: [ ns.<gatewayId> ]`).
+* Gateways now feature a customizable display name.
+  * This display name is prominently displayed throughout the API Services Portal.
+  * For existing Namespaces, the default display name will be `<IDIR>'s Gateway`.
+* The Namespace `name` attribute has been renamed to `gatewayId` for Gateways.
+  * This change aims to prevent confusion between `gatewayId` and the new `displayName` attribute for Gateways.
+  * The gatewayId is still used for setting the current Gateway in the CLI (`gwa config set gateway <gatewayId>`) and in tags within Gateway Service configuration (`tags: [ ns.<gatewayId> ]`).
+* Gateway selection user interface has changed.
+  * A new **My Gateways** page has been added, featuring a searchable list of Gateways.
+  * The Gateway dropdown selector has been relocated and updated. It now appears only on relevant API provider pages.
+* New starting page for new users after API provider login.
+  * Users without any Gateways are directed to <https://api.gov.bc.ca/manager/gateways/get-started>.
+  * This page provides a basic introduction and instructions for the new Quick Start tutorial for API providers.
+  * The new Quick Start tutorial offers a simpler option to begin with before progressing to the [Protect an API with Client Credential Flow](https://developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/tutorials/protect-client-cred/) tutorial (previously called Quick Start).
+  * The revised Quick Start tutorial utilizes a new `quick-start` configuration template.
+* Additional Changes:
+  * Added breadcrumbs on Gateway management pages.
+  * **Gateways** is no longer visible in the navigation bar for non-IDIR users (BCeID, BC Services Card, GitHub).
+  * The **Namespace Access** page is renamed to **Administration Access**.
+
+## GWA-CLI V3
```

Comment:
As per https://dev.developer.gov.bc.ca/docs/default/component/aps-infra-platform-docs/contribute/style-guide/#use-code-style-for-gwa-command-line-tool. And then I dropped to `v` to be in line with the format for API Services Portal. Changing to v on both would also be reasonable.
```suggestion
## `gwa` CLI 3.0.1
```